Project_Management_ELCA/Project_Management_ELCA/src/main/java/pilotproject/Project_Management_ELCA/entity/Project.java

```diff
@@ -0,0 +1,59 @@
+package pilotproject.Project_Management_ELCA.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+//import javax.validation.constraints.Future;
+//import javax.validation.constraints.Pattern;
+import java.time.LocalDate;
+import java.util.Set;
+
+/**
+ * @author vlp
+ */
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Project {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "group_id")
+    private Group group;
+
+    @Column(unique = true, nullable = false)
+    private int project_number;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String customer;
+
+    @Column(nullable = false)
+//    @Pattern(regexp = "^(NEW|PLA|INP|FIN)$", message = "Invalid value. Please choose from: NEW, PLA, INP, FIN")
+    private String status;
```

Comment:
Hint: which data type in Java that forces values must be in one of the declared constant?